.github/workflows/ci.yml

```diff
@@ -0,0 +1,14 @@
+name: Build
+on: [push, pull_request]
+jobs:
+  build:
+    runs-on: macos-latest
```

Comment:
Can we expand to Windows, Linux, and Mac please, and also test on JDK 8, 11, and 17 please?